lib/int_32.js

```diff
@@ -6,15 +6,15 @@ class Int32 {
   /**
    * Create an Int32 type
    *
-   * @param {number|Number} value the number we want to represent as an int32.
+   * @param {number|Number|string} value the number we want to represent as an int32.
```

Comment:
kind of looks like we accept any type here now? Since you're casting, you can convert any JSValue into a numeric